kasa/deviceconfig.py

```diff
@@ -202,9 +159,13 @@ class DeviceConfig:
     #: in order to determine whether they should pass a custom http client if desired.
     uses_http: bool = False
 
-    # compare=False will be excluded from the serialization and object comparison.
+    # compare=False will be excluded from object comparison.
     #: Set a custom http_client for the device to use.
```

Comment:
```suggestion
    #: Set a custom http_client for the device to use.
    # compare=False will be excluded from object comparison.
```
It probably makes sense to swap this? Or add the comment directly after the relevant line below.

Reply:
Removed the `compare=False` comment as this is part of the dataclass docs anyway.